packages/manager/modules/pci/src/projects/project/ai/apps/apps.html

```diff
@@ -2,6 +2,16 @@
     <cui-message-container
         data-messages="$ctrl.messages"
     ></cui-message-container>
+    <oui-message data-type="info" dismissable>
+        <span data-translate="pci_ai_app_banner_user_and_roles_info"></span>
+        <p class="m-0">
+            <a
+                rel="noopener"
+                data-translate="pci_ai_app_banner_user_and_roles_info_link"
+                data-ng-click="$ctrl.goToUsersAndTokens()"
+            ></a>
```

Comment:
Same https://github.com/ovh/manager/pull/9446/files#r1229306088

Reply:
Sorry, forget to rebase, this code was for an old P, not linked to this one.